### PR TITLE
Be/new wave numbers

### PR DESCRIPTION
**Pull request by beccaelenzil**
1) Update Wave numbers to align with Hello Books and Building an API lessons
2) Add a review and refactor wave now that 404s are introduced earlier

### CODE REVIEW

File: project-directions/wave_05.md [hunk 1]
```diff
@@ -1,30 +1,9 @@
...
-
-Create test fixtures and unit tests for the following test cases:
-
-1. `GET` `/planets/1` returns a response body that matches our fixture
-1. `GET` `/planets/1` with no data in test database (no fixture) returns a `404`
-1. `GET` `/planets` with valid test data (fixtures) returns a `200` with an array including appropriate test data
-1. `POST` `/planets` with a JSON request body returns a `201`
\ No newline at end of file
+As time allows, add custom routes. 
+* Consider using query params
```
Comment (CheezItMan) [inline project-directions/wave_05.md:9]: Do we let them decide on custom routes?

File: project-directions/wave_06.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Wave 06: Writing Tests
```
Comment (CheezItMan) [inline project-directions/wave_06.md:1]: Just asking should we do tests step-by-step as we build the app, TDD fashion or save tests for the end?

**Review by CheezItMan — APPROVED** [review]
:shipit:

File: project-directions/wave_06.md (hunk 2 above)
Comment (beccaelenzil) [reply project-directions/wave_06.md:1]: currently we don't teach tests until after teaching all the operations, so I think we have to save them for the end in this case

**Pull request merged**